src/pkg/services/m365/api/config.go

```diff
@@ -26,6 +26,7 @@ const (
 	isCancelled       = "isCancelled"
 	isDraft           = "isDraft"
 	mobilePhone       = "mobilePhone"
+	modifiedTime      = "lastModifiedDateTime"
```

Comment:
Nit: I'm trying to keep these var names parallel to their values to minimize cognitive load.
```suggestion
	lastModifiedDateTime      = "lastModifiedDateTime"
```